### PR TITLE
Implement dormitory pictures logic in backend

### PR DESCRIPTION
**Pull request by sergiynikon**

### CODE REVIEW

**Review by RomanNakonechnyi — APPROVED** [review]
Дуже гарний пул ріквест

File: src/backend/LDW.Application/Features/DormitoryFeatures/Commands/UploadDormitoryPictureCommand.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+using System.Threading.Tasks;
+
+namespace LDW.Application.Features.DormitoryFeatures.Commands
+{
+
+	public class UploadDormitoryPictureCommand : IRequest<DormitoryPictureModel>
+	{
+		public int DormitoryId { get; set; }
+		public bool IsMain { get; set; }
+		public byte[] Image { get; set; }
```
Comment (OlesiaTvardovska) [inline src/backend/LDW.Application/Features/DormitoryFeatures/Commands/UploadDormitoryPictureCommand.cs]: Do we save images in db? We have already configured azure blob container (e.g for saving user profile pics). Adding picture to some dormitory, we could just save URL of the image to the db.

**Review by OlesiaTvardovska — APPROVED** [review]

File: src/backend/LDW.Application/Features/DormitoryFeatures/Commands/UploadDormitoryPictureCommand.cs (hunk 1 above)
Comment (sergiynikon) [reply src/backend/LDW.Application/Features/DormitoryFeatures/Commands/UploadDormitoryPictureCommand.cs]: oh, did't know, Will change

Comment (OlesiaTvardovska) [reply src/backend/LDW.Application/Features/DormitoryFeatures/Commands/UploadDormitoryPictureCommand.cs]: I guess it's ok for now)

Comment (sergiynikon) [reply src/backend/LDW.Application/Features/DormitoryFeatures/Commands/UploadDormitoryPictureCommand.cs]: i already changed :)

**Pull request merged**